source/agora/common/Config.d

```diff
@@ -103,6 +103,9 @@ public struct NodeConfig
     /// Is this a validator node
     public bool is_validator;
 
+    /// There is enrollment data in the genesis block
+    public bool has_enroll = true;
```

Comment:
This does not belong here. This is the Node's public configuration. Why is there a workaround for a unittest in the Node configuration?

Reply:
Okay